js/infobar.js

```diff
@@ -33,9 +38,7 @@
             if (sessionStorage.getItem(this.prefName) === 'true') {
                 this.disabled = true;
             }
-        } catch (ex) {
-            console.log('Error thrown while getting disabled state from sessionStorage: ', ex);
-        }
+        } catch (ex) {}
 
         // If there is already another $infoBar, don't show this
         if (config.$infoBar.filter(':visible').length) {
```

Comment:
Small performance suggestion - maybe change to `if (!this.disabled && config.$infoBar.filter(':visible').length)` to avoid the DOM hit if already disabled in `sessionStorage`?